src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -31,6 +30,8 @@
 import netaddr
 import requests.packages
 
+import discover_nodes.dracclient.client as discover_nodes_dracclient
```

Comment:
Same here.  I don't understand why we are switching to DRACClient in discover_nodes.

Reply:
As stated in the above comment this change is also done for the same purpose.
Removed upstream dracclient and reverted back downstream dracclient (same as master).
This changes eventually point to https://github.com/dsp-jetpack/JetPack/blob/8cff5fd7c2ed90876d4cf851862a30bf756480d7/src/pilot/discover_nodes/discover_nodes.py#L390 and https://github.com/dsp-jetpack/JetPack/blob/8cff5fd7c2ed90876d4cf851862a30bf756480d7/src/pilot/discover_nodes/discover_nodes.py#L391

Earlier this change is done via newly imported upstream dracclient, but now it is working from downstream via inheritance.